.retest/recheck.ignore.js

```diff
@@ -0,0 +1,46 @@
+var allowedPxDiff = 5;
+var baseUrl = /http[s]?:\/\/[\w.:\d\-]*/;
+var fontFamilies = [["system-ui", "Arial"], ["-apple-system", "sans-serif"]];
+
+function contains(array, key) {
```

Comment:
And again: I'm not that familiar with JS, but … 😊 Can't we use [`Array.includes()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes)? According to [MDN](https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects), these objects and methods are always available in any JS environment? If this is true, maybe we can simplify more code?

Reply:
Maybe nashorn doesn't have it? `["absolute-x", "absolute-y", "absolute-width", "absolute-height", "x", "y", "width", "height"].includes is not a function in <eval> at line number 28
22:33:07.798 [main] INFO de.retest.recheck.ignore.JSShouldIgnoreImpl - Reading JS ignore rules file from .retest/recheck.ignore.js.`